commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/LineItemDraft.java

```diff
@@ -17,13 +17,17 @@
 @JsonDeserialize(as = LineItemDraftDsl.class)
 @ResourceDraftValue(
         abstractBuilderClass = true,
-        factoryMethods = {@FactoryMethod(parameterNames = {"productId"})})
+        factoryMethods = {
+                @FactoryMethod(methodName = "ofSku", parameterNames = {"sku", "quantity"}),
```

Comment:
i didn't understand why the quantity attribute was added for the factory method 🤔